Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -38,6 +38,10 @@ class MessageDTO: NSManagedObject {
     @NSManaged var replyCount: Int32
     @NSManaged var extraData: Data?
     @NSManaged var isSilent: Bool
+
+    // Used for creating a message as a system message from the client SDK.
+    @NSManaged var isSystem: Bool
```

Comment:
@martinmitrevski I've removed the `isSystem` prop. The reason I added this, was because when using the message for the HTTP Request, we should only send the type if it is a system message, but now I check if the type is system, and if yes, then we send it. So the `isSystem` is not really needed 👍